src/main/java/com/google/sps/data/User.java

```diff
@@ -0,0 +1,13 @@
+package com.google.sps.data;
+
+import java.util.HashSet;
+
+public final class User {
+  private final String email;
+  private final HashSet<Long> groups;
+
+  public User(String email) {
+    this.email = email;
+    this.groups = new HashSet<>();
```

Comment:
You can do this when you instantiate the variable above since it doesn't depend on any passed-in data